apps/glusterfs/app.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/boltdb/bolt"
 	"github.com/gorilla/mux"
 	"github.com/heketi/heketi/executors"
+	"github.com/heketi/heketi/executors/caiexec"
```

Comment:
:accept: see ce5f8a8179d53a7a20c908e2ed3f4a49eb82006f and e31d8aed6bf2b50be65914c88a6468eb4319846a